src/hooks/useLoginApi.ts

```diff
@@ -80,15 +80,19 @@ const useLogoutApi = () => {
     []
   );
 
-  const callbackSuccess = useCallback(() => {
+  const callbackSuccess = useCallback((localStorageTheme?: string | null) => {
     cookies.remove(`${process.env.REACT_APP_AUTH_COOKIE}`);
     document.location.replace('./');
+    if (localStorageTheme) {
+      localStorage.setItem('discovery-ui-theme', localStorageTheme);
```

Comment:
As discussed in Slack here (https://redhat-internal.slack.com/archives/C02QSNF1UKE/p1757083758912669?thread_ts=1757017774.746129&cid=C02QSNF1UKE), to satisfy prodsec an follow best security practices, when the user wants to log out, we should always and only clear all cookies and local storage, and we should _not_ write back into them. I'm not sure if that requirement kills this whole PR, but at the very least, I think we should not add this logic here to `useLogoutApi` because we should not be setting any more/new items in `localStorage` here.